app/src/main/java/org/fossasia/openevent/app/common/livedata/ActionLiveData.java

```diff
@@ -0,0 +1,47 @@
+package org.fossasia.openevent.app.common.livedata;
+
+import android.arch.lifecycle.LifecycleOwner;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.Observer;
+import android.support.annotation.MainThread;
+import android.support.annotation.Nullable;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * A wrapper over MutableLiveData in order to send a one time action from ViewModel.
+ * @param <T> Data type of the LiveData.
+ */
+public class ActionLiveData<T> extends MutableLiveData<T> {
```

Comment:
Is it different from SingleEventLiveData

Reply:
No, do we have a SingleEventLiveData already in the project ?

Reply:
No, it should be renamed to that and also Hungarian notation should be changed

Reply:
Oh, alright.